test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -6810,6 +6810,8 @@ public async Task RestoreNetCore_SingleTFM_SimplePackageDownload()
                 solution.Projects.Add(projectA);
                 solution.Create(pathContext.SolutionRoot);
 
+                Util.CreateTempGlobalJson(pathContext.SolutionRoot);
```

Comment:
Can this be added in some intermediate function that's used everywhere, like the SimpleTestPathContext?

This feels like it doesn't scale, as it's likely that people might forget to add it.

Reply:
This is a temporary workaround since the VS on CI could not resolve the preview SDK. So the nuget restore on SDK style project would fail. It will be removed when VS on CI upgrade or SDK deliver the RTM version.
The other global.json creation function is in msbuildIntegratedFixture. Since only SDK3.0 would be patched, there is no need to specify which SDK to use. It's going to be removed as well.